### DISCUSSION TITLE
팀원소개

### DISCUSSION THREAD

**Discussion by thrcle** [Pseudo-Lab/healthy_dessert_recommender_system#1] [Announcements]
9th 가짜연구소 <'내게 딱 맞는 건강디저트 추천시스템 구현하기> 프로젝트에 참여하는 팀원을 소개하는 페이지입니다 🍀

**Comment by thrcle** [comment]
<img width="140" alt="image" src="https://github.com/user-attachments/assets/d3f3bbc2-83fa-4b15-9ed3-39949b2f57f7"><p>

[심세원] ([링크드인](http://www.linkedin.com/in/se-won-sim-621a46280))
프로젝트명:내게 딱 맞는 건강디저트 추천시스템 구현하기

### 인삿말
안녕하세요. ‘변하고싶으면 환경을 바꿔라’는 말을 듣고 성장할 수 밖에 없는 환경을 찾다가 가짜연구소 문을 두드린 심세원입니다. 🚪✊

### 간단한 이력
저는 현재 식품기업 데이터&AI팀에서 1년 9개월정도 재직중입니다.
수행한&진행중인 주요 업무는 다음과 같습니다.
    - NPD(New Product Development Process) 개선을 위한 CX(Customer Experience) 관리 플랫폼 기획,시스템 설계, 리뷰요약 서비스 구축 참여 (ing)
    - 자사&경쟁사 리뷰데이터 분석을 통한 제품 특성 분석-개선점 도출
    - 전국 H마트 VMI(Vender Management Inventory) 예측모델링 운영 
    - CVS채널 수요예측 운영


### 관심사
호기심이 많아서 다양하게 탐색하는 것을 좋아합니다. 다양한 가치관을 가진 사람들의 이야기&경험을 듣는 것도 좋아합니다.

1) 커리어
 커리어 패스를 어떻게 가져가면 좋을지에 대한 관심&고민이 많습니다. 아직 저년차인만큼 좁고 깊은 전문성 부분은 앞으로 부지런히 채울 예정입니다.
2) 사이드 프로젝트
회사 일말고 내가 관심있는 분야에서 역량을 펼쳐보고 싶다는 생각이 들어 현재 사이드 프로젝트로 건강디저트 플랫폼 운영전략, 데이터부분(GA4, 추천시스템 구현)에서 기여하고 있습니다. 
3) 취미
취미로 펜싱, 테니스, 백패킹을 (찍먹정도) 해본 적이 있습니다
귀여운 것을 너무 좋아해서 휴가내고 평일에 서일페 가보기, 인스타툰 연재해보기가 버킷리스트입니다

### 특별한 경험
- 저는 정리/방향을 이끌고, 인내심&책임감을 발휘해야하는 상황에 강합니다. 
- 대학생 시절 지역아동센터에서 교육활동을 주 6시간씩 1년 반정도 멘토링한적이 있습니다. 책에 대한 거부감이 아주 심한 아이를 집중 멘토링해서 6개월 뒤에 스스로 독후감까지 쓸수 있도록 변화시켜본 경험이 있습니다. (우수 멘토링 사례로도 뽑혀서 아주 뿌듯했던 순간이었습니다)
- 취준시절 멋쟁이사자처럼 X 쏘카 AI 엔지니어 부트캠프 2기에 참여하여 스터디를 리딩 +. Vision 기술을 쏘카서비스에 접목시킬 수 있는 과제를 기획해본 적이 있습니다. (실제 서비스에 적용까지는 못했고, 기술 구현 demo까지만 수행) 
- ‘글또’라는 개발자 글쓰기 커뮤니티에서 6개월동안 12개의 포스팅 미션을 한번도 패스를 사용하지 않고 끝까지 완수해서 예치금 환급+추가금 까지 받은 경험이 있습니다.

**Comment by zzakjista** [comment]
<img src="https://github.com/user-attachments/assets/d04538fd-2cc3-4dc4-bf97-c330dcaf09f1" width="200" height="200"/>

[박지훈] ([링크드인](https://www.linkedin.com/in/%EC%A7%80%ED%9B%88-%EB%B0%95-0a13b5262/))
## 인삿말
안녕하세요. 박지훈입니다. 저는 소소한 꿈이 있는데요. 나이가 많이 들어서도 말랑한 생각들을 내면화하며 사는 삶을 사는 것입니다.
그런 소망의 일환으로 이 곳에 닿게 되었네요. 

## 이력
현재는 넥슨 코리아의 인텔리전스랩스의 추천팀에서 근무하고 있습니다. 게임 속 유저들의 몰입과 흥미를 높이는 일들을 하고 있습니다.
제 본 전공은 광고학 전공이었으나, 현생과 이상에서 타협을 하다보니 어느새 데이터를 만지고 있는 사람이 되었네요. 
현재 직장에선 추천 서비스 기획 및 개발, 라이브(운영)하고 효과를 분석하는 일들을 하고 있어요.

## 관심사
찌르면 기름이 나올 것만 같은 로봇은 되기가 싫어서요. 많은 자극을 들이고 내보내며 살려고 하고 있습니다.
- 독서를 합니다. 철학을 배우길 좋아합니다. 최근에는 자유론을 읽었어요.
- 게임도 합니다. 컨트롤러를 사서 재밌게 콘솔 게임 하고 있어요.
- 주말마다 개발 공부도 합니다. 일하다 보니 제가 부족한 점을 많이 느끼게 되네요. 

## 특별한 경험
학부 때 커리어를 크게 바꾼 것이 좀 유별난 선택이었던 것 같아요.
돌아보니 아직도 2년 밖에 되지 않았네요!

## 취미
이야기 나누는 것과 글쓰기를 좋아합니다.
얘기가 잘 통하면 기분이 좋아요.

**Comment by Hugh-Kang** [comment]
![image](https://github.com/user-attachments/assets/07d94b2f-e36c-46da-9f3b-8890d0bd7316)

[강해우] ([링크드인](https://www.linkedin.com/in/%ED%95%B4%EC%9A%B0-%EA%B0%95-476814256/))
프로젝트명:내게 딱 맞는 건강디저트 추천시스템 구현하기

인삿말
: 안녕하세요. ‘딴짓을 좋아하는 서비스 기획자' 강해우입니다!

간단한 이력
- 챗봇 AI 스타트업에서 전문연구요원으로 대체 복무중입니다. 올해 12월이면 끝나네요(감격.)
- 개발자를 거쳐서, 데이터 싸이언티스트를 하다가 현재는 서비스 기획과 PM 직무를 하고 있습니다.
- 석사 전공은 UX, HCI, Data Science이라서, 돌고 돌아 서비스 기획을 하게 되었어요.
- 참고로 빵은 잘 안 먹는데, 빵 구경을 좋아합니다

관심사
- 최근에 직장인반 뮤지컬을 성공적으로 잘 마쳤습니다. 노래/연기는 못하지만 살면서 한번쯤은 해볼만한 취미더라구요.
- 아침에 월수금 수영을 가는데, 어깨가 넓어지기 보다는 건강한 돼지가 되는 기분입니다.
- 여행을 좋아해서 친구들과 국내/국외 여행을 자주 갑니다. 유명한 여행지는 많이 가본 것 같은데, 새로운 여행지를 추천받고 싶어요!

특별한 경험
- 호주에서 잠깐 살 때, 매주 노숙자 봉사를 한 적이 있는데, 올해 10월 Melbourne여행에서 당시 친구들과 다시 봉사할 예정입니다.
- 공대 출신이지만, 올해 여름에 한국어 교원 자격증 2급을 취득했고, (언젠가)외국인 대상으로 한국어 교육 봉사를 하고 싶습니다..

**Comment by psm981021** [comment]
![박성범 (1)](https://github.com/user-attachments/assets/e53de4fb-9a50-4488-92e9-e52318727d12)



### 인삿말
안녕하세요, 도전을 즐겨하는 박성범입니다! 새로운 도전을 하고자 가짜연구소를 찾게 되었습니다..!

### 간단한 이력
세종대학교 데이터사이언스학과 재학 중이며, 학부 연구생으로 추천 시스템과 관련해서 연구를 진행 중입니다.

### 관심사
개인화 추천시스템에 큰 관심이 있으며, 추천 시스템에서 LLM의 도입에도 관심이 있습니다.

### 특별한 경험

고소공포증이 있지만 암벽 등반을 해본 경험이 있습니다!

**Comment by ksj1368** [comment]
장성준([링크드인](https://www.linkedin.com/in/%EC%84%B1%EC%A4%80-%EC%9E%A5-12a601205/))

## 인삿말

안녕하세요, 장성준입니다.  

## 이력

- 주전공은 공간정보공학, 복수전공으로 통계학을 전공했습니다. 올해 2월에 졸업한 후 현재는 취업을 준비하고 있습니다.

- 주전공과 관련된 대학원 진학을 고려했으나 한 가지 주제를 오랜 기간 연구하는 것이 저와 맞지 않다고 판단해 데이터 분야로 진로를 변경하게 되었습니다.

## 관심사

- 최근에는 바이럴 탐지 프로젝트를 진행하고 있어 자연어 이상탐지 관련 자료를 찾아보고 있습니다.

- 다양한 데이터를 다뤄볼 수 있는 대회에 참여하는 것을 좋아합니다.

- 게임과 음악을 좋아합니다. 요즘은 데드락이라는 게임을 주로 하고 있습니다.
